frontend/src/components/Chessboard.vue

```diff
@@ -113,7 +119,11 @@ export default {
     },
     loadPosition() {
       this.game = new Chess();
-      this.game.load(this.fen);
+      if (!window.localStorage.getItem("AIfen")) {
+        this.game.load(this.fen);
+      } else {
+        this.game.load(window.localStorage.getItem("AIfen"));
```

Comment:
This isn't OK, because Chessboard.vue is parent for both Humanboard and Aiboard. 
Let's pass the history as a prop here.